src/nyc_trees/nyc_trees/settings/base.py

```diff
@@ -286,3 +286,5 @@
 # If geocoding a string produces no results, this string will be
 # appended for a second attempt.
 GEOCODE_FALLBACK_SUFFIX = ', New York, NY'
+
+TILER_URL = environ['TILER_HOST']
```

Comment:
I struggled with this.  I don't like providing default values in 2 places, and there is already a default in the Ansible playbooks.  I can add it here too though.  (FYI, the default would be `localhost:7000`, not `localhost`)